be/src/http/action/greplog_action.cpp

```diff
@@ -66,7 +66,7 @@ void GrepLogAction::handle(HttpRequest* req) {
         return;
     }
 
-    auto ret = grep_log_as_string(start_ts, end_ts, std::toupper(level[0]), pattern, limit);
+    auto ret = grep_log_as_string(start_ts, end_ts, level, pattern, limit);
```

Comment:
before the change, suppose level can be lower case, the function is still correct because of toupper. After this change, is it still true? I didn't see the case conversion or case-insensitive search.

Reply:
done